scripts/data_access/create_data_access_view.sql

```diff
@@ -0,0 +1,132 @@
+                      'storage.objects.update',
+                      'storage.buckets.create',
+                      'storage.buckets.delete',
+                      'storage.buckets.get',
+                      'storage.buckets.getIamPolicy',
+                      'storage.buckets.list',
+                      'storage.buckets.setIamPolicy',
+                      'storage.buckets.update') THEN
+        'ADMIN'
+      ELSE
+        CONCAT('Unknown storage:', action)
+      END
+    -- See granular permissions here: https://cloud.google.com/bigquery/docs/access-control#bq-permissions
+    WHEN service = 'bigquery' THEN
+      CASE
+      WHEN action IN ('bigquery.tables.delete',
+                            'bigquery.datasets.delete',
+                            'bigquery.jobs.update',
+                            'bigquery.routines.delete',
+                            'bigquery.tables.updateData') THEN
+        'WRITE'
+      WHEN action IN ('bigquery.tables.getData',
+                            'bigquery.tables.export',
+                            'bigquery.readsessions.create',
+                            'bigquery.connections.use') THEN
+        'READ'
+      WHEN action IN ('bigquery.jobs.create',
+                            'bigquery.jobs.listAll',
+                            'bigquery.jobs.list',
+                            'bigquery.jobs.get',
+                            'bigquery.datasets.create',
+                            'bigquery.datasets.get',
+                            'bigquery.datasets.update',
+                            'bigquery.tables.create',
+                            'bigquery.tables.list',
+                            'bigquery.tables.get',
+                            'bigquery.tables.update',
+                            'bigquery.routines.create',
+                            'bigquery.routines.list',
+                            'bigquery.routines.get',
+                            'bigquery.routines.update',
+                            'bigquery.transfers.get',
+                            'bigquery.transfers.update',
+                            'bigquery.savedqueries.create',
+                            'bigquery.savedqueries.get',
+                            'bigquery.savedqueries.list',
+                            'bigquery.savedqueries.update',
+                            'bigquery.savedqueries.delete',
+                            'bigquery.connections.create',
+                            'bigquery.connections.get',
+                            'bigquery.connections.list',
+                            'bigquery.connections.update',
+                            'bigquery.connections.delete') THEN
+        'ADMIN'
+      ELSE
+        CONCAT('Unknown bigquery:', action)
+      END
+    ELSE
+      CONCAT('Unknown service:', service)
+  END AS op,
+  granted,
+  -- Project is of the resource or, if not there,
+  -- then for the method accessing it (eg for buckets)
+  CASE
```

Comment:
Do you need a ELSE to set a default value here? Otherwise, it will create NULLs.